02-basics-2/50-selected-meetup/SelectedMeetupApp.js

```diff
@@ -1,75 +1,64 @@
-import { defineComponent } from 'vue'
-// import { getMeetup } from './meetupsService.ts'
+import { defineComponent, ref, watch } from 'vue'
+import { getMeetup } from './meetupsService.ts'
 
 export default defineComponent({
   name: 'SelectedMeetupApp',
 
-  setup() {},
+  setup() {
+    const numbers = ref([1, 2, 3, 4, 5])
+    const title = ref('')
+    const selectedId = ref(numbers.value[0])
+
+    watch(async () => {
+      if (selectedId.value) {
+        const data = await getMeetup(selectedId.value)
+        title.value = data.title
+      }
+    })
```

Comment:
This is still not a valid `watch` usage

Reply:
sorry, it's my fail, I didn't update the corrected solution to this task